.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Deploy Docs
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-20.04
+    env:
+      MIKE_VERSION: v3
+      MIKE_ALIAS: latest
+      DOCKER_IMAGE: squidfunk/mkdocs-material-modified:8.5.9
```

Comment:
I still find the image name confusing. It starts with `squidfunk/` so at first look it seems to be a thirdparty image.
I'd suggest to use `ui5-tooling/mkdocs-material`.

Reply:
What about the version? Would you prefer to keep the version aligned with the original image or to omit it and always be on the "latest"?

Reply:
I think we don't need a version or tag at all, as we always re-build the image.